infrastructure/sentry-subgraph/sepolia.subgraph.yaml

```diff
@@ -135,6 +135,26 @@ dataSources:
         - event: ConvertedToEsXai(indexed address,uint256)
           handler: handleConvertedToEsXai
       file: ./src/xai.ts
+  - kind: ethereum
+    name: Rollup
+    network: arbitrum-sepolia
+    source:
+      address: "0xC47DacFbAa80Bd9D8112F4e8069482c2A3221336" #TODO: change to sepolia rollup address when deployed in a later ticket
+      abi: Rollup
+      startBlock: 169689813 #TODO: change to deploy block number when rollup contract is deployed in a later ticket
```

Comment:
Do we have tickets created for this? If not, you should make a placeholder for a reminder to discuss in IPM.

Reply:
Added a [ticket](https://www.pivotaltracker.com/story/show/188211050) to update these values after sepolia deployment.